solutions/USACO dec17-milk-measurement.mdx

```diff
@@ -0,0 +1,141 @@
+---
+id: usaco-592
+title: USACO December 2017 Bronze - The Bovine Shuffle
+author: Jesse Choe
+---
+
+[Official Analysis]: http://www.usaco.org/current/data/sol_measurement_bronze_dec17.html
```

Comment:
There's already an official C++ implementation; is there a particular reason to include another one?